src/components/search/SearchResults.tsx

```diff
@@ -216,93 +241,104 @@ const SearchResults: FunctionComponent<SearchResultsProps> = ({ serverId = windo
-                cardOptions={{ showYear: true }}
-            />
-            <SearchResultsRow
-                title={globalize.translate('Episodes')}
-                items={episodes}
-                cardOptions={{
-                    coverImage: true,
-                    showParentTitle: true
-                }}
-            />
-            <SearchResultsRow
-                title={globalize.translate('HeaderVideos')}
-                items={videos}
-                cardOptions={{ showParentTitle: true }}
-            />
-            <SearchResultsRow
-                title={globalize.translate('Programs')}
-                items={programs}
-                cardOptions={{
-                    preferThumb: true,
-                    inheritThumb: false,
-                    showParentTitleOrTitle: true,
-                    showTitle: false,
-                    coverImage: true,
-                    overlayMoreButton: true,
-                    showAirTime: true,
-                    showAirDateTime: true,
-                    showChannelName: true
-                }}
-            />
-            <SearchResultsRow
-                title={globalize.translate('Channels')}
-                items={channels}
-                cardOptions={{ shape: 'square' }}
-            />
-            <SearchResultsRow
-                title={globalize.translate('Playlists')}
-                items={playlists}
-            />
-            <SearchResultsRow
-                title={globalize.translate('Artists')}
-                items={artists}
-                cardOptions={{ coverImage: true }}
-            />
-            <SearchResultsRow
-                title={globalize.translate('Albums')}
-                items={albums}
-                cardOptions={{ showParentTitle: true }}
-            />
-            <SearchResultsRow
-                title={globalize.translate('Songs')}
-                items={songs}
-                cardOptions={{ showParentTitle: true }}
-            />
-            <SearchResultsRow
-                title={globalize.translate('HeaderPhotoAlbums')}
-                items={photoAlbums}
-            />
-            <SearchResultsRow
-                title={globalize.translate('Photos')}
-                items={photos}
-            />
-            <SearchResultsRow
-                title={globalize.translate('HeaderAudioBooks')}
-                items={audioBooks}
-            />
-            <SearchResultsRow
-                title={globalize.translate('Books')}
-                items={books}
-            />
-            <SearchResultsRow
-                title={globalize.translate('Collections')}
-                items={collections}
-            />
-            <SearchResultsRow
-                title={globalize.translate('People')}
-                items={people}
-                cardOptions={{ coverImage: true }}
-            />
+            {isLoading ? (
+                <div className='loading-circle'></div>
```

Comment:
You could probably use the `<Loading />` component for this?

https://github.com/jellyfin/jellyfin-web/blob/master/src/components/loading/LoadingComponent.tsx

It will show the global loading component we use in other places.

Reply:
Oh I didn't even know that existed! I'll switch that over.